README.md

```diff
@@ -7,31 +7,43 @@ in a uniform way.
 Installation
 ------------
 Serviced can run on a single host or across multiple hosts. First, serviced
-depends on docker.
+depends on docker. The minimal version of docker should be **19.03.11** for running serviced **1.8.x**.
+The ubuntu version should be **18.04** or newer
 
-1. Follow the instruction at http://www.docker.io/ , to install
+1.  Follow the instruction at http://www.docker.io/ , to install
    it on every host that serviced will run on. Ensure docker is running.
 
-2. Install a generated package
+2.  Install a generated package
+      ```bash 
+          sudo apt-key adv --keyserver keys.gnupg.net --recv-keys AA5A1AD7
+    ￼     REPO=http://get.zenoss.io/apt/ubuntu
+          sudo sh -c 'echo "deb [ arch=amd64 ] '${REPO}' trusty universe" > /etc/apt/sources.list.d/zenoss.list'
+          sudo apt-get update
+          sudo apt-get install -y serviced
+      ```
 
-      sudo apt-key adv --keyserver keys.gnupg.net --recv-keys AA5A1AD7
-￼     REPO=http://get.zenoss.io/apt/ubuntu
-      sudo sh -c 'echo "deb [ arch=amd64 ] '${REPO}' trusty universe" \
-          > /etc/apt/sources.list.d/zenoss.list'
-      sudo apt-get update
-      sudo apt-get install -y serviced
+    Or follow the steps below in the dev section to a source build.
+   
+3.  Elasticsearch 7.x uses a `mmapfs` directory by default to store its indices. 
+    The default operating system limits on mmap counts is likely to be too low, 
+    which may result in out of memory exceptions. 
+    On Linux, the limits should be increased by running the following command as root:
+    ```bash
+    sysctl -w vm.max_map_count=262144
+    ```
+   
+       To set this value permanently, update the `vm.max_map_count` setting in `/etc/sysctl.conf`. 
+       To verify after rebooting, run `sysctl vm.max_map_count.`
```

Comment:
Adding a file to `/etc/sysctl.d/` will also work.
```bash
$ sudo echo "vm.max_map_count=262144" > /etc/sysctl.d/10-vm.conf
```

Reply:
It's an official suggestion from the elastic site. I assume this more general approach. 
https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html

Reply:
My suggestion could be more general.  The /etc/sysctl.d directory allows multiple users/packages from making changes to the same file.  The /etc/sysctl.d directory also supports priority.

Since what you wrote is copied from the elasticsearch website, I'll let it stand as is.